config/locales/cy.yml

```diff
@@ -102,6 +102,15 @@ cy:
   components:
     figure:
       image_credit: 'Delwedd gan: %{credit}'
+  content_item:
+    schema_name:
+      take_part:
```

Comment:
There's also a `formats` lower this file that contains content for the original frontend document types.

I think they should all be in the same place, wherever that is.

Reply:
Moved to formats